chalicelib/app_utils.py

```diff
@@ -131,12 +131,15 @@ def check_authorization(request_dict, env=None):
                 return False  # we have no env to check auth
             # leeway accounts for clock drift between us and auth0
             payload = jwt.decode(token, b64decode(auth0_secret, '-_'), audience=auth0_client, leeway=30)
-            env_info = init_environments(env)
-            user_res = ff_utils.get_metadata('users/' + payload.get('email').lower(),
-                                            ff_env=env_info[env]['ff_env'], add_on='frame=object')
-            if 'admin' in user_res['groups'] and payload.get('email_verified'):
-                # fully authorized
-                return True
+            envs = env.split(',') # handle multiple envs
```

Comment:
"all" is a valid foursight environment, which will not work with the code as written. Easy fix:
```
# will handle "all" as well as comma separated envs
for environment in init_environments(env):
```

Reply:
Fixed and now being tested